### PR TITLE
azure resource comboboxes

### PR DESCRIPTION
**Pull request by wangmingliang-ms**

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/appservice/Platform.java [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+public interface Platform {
+
+    OperatingSystem getOs();
+
+    String getStackOrWebContainer();
+
+    String getStackVersionOrJavaVersion();
+
+    List<Platform> platforms = ImmutableList.copyOf(new Platform[]{
```
Comment (Flanker32) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/appservice/Platform.java:17]: Shall we move the value to a constant class?

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/appservice/intellij/view/component/AzureComboBox.java [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+        this.setRenderer(new SimpleListCellRenderer<T>() {
+            @Override
+            public void customize(final JList l, final Object o, final int i, final boolean b, final boolean b1) {
+                setText(getItemText(o));
+                setIcon(getItemIcon(o));
+            }
+        });
+    }
+
+    public void refreshItems() {
```
Comment (Flanker32) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/appservice/intellij/view/component/AzureComboBox.java:49]: we may add an overload function to set the default value

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/appservice/Platform.java (hunk 1 above)
Comment (wangmingliang-ms) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/appservice/Platform.java:17]: No, I don't think so. if you want to find platforms, go to `Platform`.

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/appservice/intellij/view/component/ComboBoxServicePlan.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+    public void refreshWith(Subscription subscription) {
+        this.subscription = subscription;
+        this.refreshItems();
+    }
+
+    @NotNull
+    @Override
+    protected List<? extends AppServicePlan> loadItems() throws Exception {
+        if (Objects.nonNull(this.subscription)) {
```
Comment (Flanker32) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/appservice/intellij/view/component/ComboBoxServicePlan.java:35]: We may need to filter runtime/os as well

**Review by Flanker32 — APPROVED** [review]

**Pull request merged**